src/prog_models/prognostics_model.py

```diff
@@ -1344,7 +1344,13 @@ def default(self, o):
             elif isinstance(o, DictLikeMatrixWrapper):
                 dict_temp = {k: v for k, v in o.items()}
                 dict_temp['_original_type'] = 'DictLikeMatrixWrapper'
-                return dict_temp 
+                return dict_temp
+            elif isinstance(o,np.bool_):
+                if o == False:
```

Comment:
It might be cleaner to use casting. For example:

return bool(o) 
instead of line 1349-1353

Reply:
Great suggestion, made the edit